modules/ROOT/pages/fd-deploy-app-from-rm.adoc

```diff
@@ -1,15 +1,18 @@
 = Deploy a Mule Application from Runtime Manager
 
-It might be the case that you are not allowed to deploy an application directly from Flow Designer. Your organization might require someone with the proper authority to deploy applications from Runtime Manager. You can share with that person a copy of your project as a `.jar` file to deploy your application with.
+It might be the case that you are not allowed to deploy an application directly from Flow Designer.
+Your organization might require someone with the proper authority to deploy applications from Runtime Manager.
+You can share with that person a copy of your project as a `.jar` file to deploy your application with.
```

Comment:
Suggest us saying "In that scenario Flow Designer allows you to export the project as a `.jar` file and share it with those who are responsible for app deployment"

Reply:
Rewritten to accomodate.